project_one/project1cmp464.html

```diff
@@ -0,0 +1,216 @@
+                    </form>
+                    </p>
+                 </div>
+                 <script>
+                 var checktheStatus = document.getElementsByName("testthree1")[0].value;
+                 var status;
+                 if(checktheStatus==on){
+                    status=document.getElementById("on").value;
+                 }
+                 else{
+                    status=document.getElementById("off").value;
+                 }
+                 
+                 </script>
+                 <script>
+                     function clickme1(){
+                     document.body.style.backgroundColor = "PaleTurquoise";
+                     }
+                     function clickme2(){
+                        document.body.style.backgroundColor = "Lavender";
+                     }
+                 </script>
+
+
+
+
+
+           
+    <script>
+
+
+  
+    </script>
+    <p style="position:absolute; left:20px; top:15%"><b>These links are show the answer of the grading-elements:</b>
+
+
+
+
+    </p>
+         <a href="project1cmp464(3).html" style="position:absolute; left:20px; top:20%">click the link to see the grading-element-one
+            </a>
+       <a href="project1cmp464(4).html" style="position:absolute; left:20px; top:23%">click the link to see the grading-element-three
+       </a>
+       <a href="project1cmp464(2).html" style="position:absolute; left:20px; top:26%">Click here to see the 4th page of the extra credit assignment</a>
+
+
+<div name="grading-element-four">
+    <p class="paragraph"><b>Search your Visiting requirement in here & we will give you some suggestion.
+    please fill all the option</b></p>
+        <form action="http://localhost:3000/form" method="post" name="grading-element-four" onsubmit="return validation()">
+           
+            <select name="testOne"  class="range">
+                    <option value="select">please select one option</option>
+                    <span name="enter"></span>
+            <option value="one">$10000-$20000</option>
+            <option value="two">$5000-10000</option>
+           
+            </select>
+            <div class="testthreeradio">
+                    <input type="radio" name="testThree" value="none" id="none" checked> no selection<br>
+            <input type="radio" name="testThree" value="Air"  id="Air"> Air<br>
+  <input type="radio" name="testThree" value="Bus" id="Bus"> Bus<br>
+  <input type="radio" name="testThree" value="Train" id="Train" > Train
+ </input>
+  <span name="radioselect"></span>
+</div>
+        
+        <input type ="text" placeholder="type here" name="testTwo" class="css42"/>
+        <span name="enter_your_search"></span>
+         <input type="submit" name="submit" class="css44" ></input>
+    
+        </form>
+    </div>
+   
+ <script type="text/javascript"> 
+ function validation(){ 
+ var testtwo = document.getElementsByName("testTwo")[0].value;
```

Comment:
validation is great! But for the test, I want to be able to submit _without making a selection_ and still see values in the form...because you set default values with javascript. You have a `testOne`, `testTwo` and `testThree` in the submitted form, though, so you're very very close!